### PR TITLE
Use MiniLZO v2.10

### PR DESCRIPTION
**Pull request by Octoate**
I updated the source to use the version 2.10 of MiniLZO instead of v1.06. It features speed and portability improvements. The version was tested under Windows and Linux with the examples of KallistOS and a bitrate of 1.5Mbps. The newly compiled version is also backward compatible with previous versions.
I also increased the version number of dctool-serial to v1.1.0.

### CODE REVIEW

**Review by ljsebald — CHANGES_REQUESTED** [review]
My apologies for how long it took to get this PR reviewed. I'd be happy to accept it, except for changing the version number. If you don't mind reverting the changes to the program's version number, then I'll be happy to merge this PR.

Thanks for your contribution!

File: Makefile.cfg [hunk 1]
```diff
@@ -70,7 +70,7 @@ endif
 # You generally shouldn't change this unless you are making forked
 # versions (or test versions)
 # Version numbers must be of the form x.x.x
-VERSION = 1.0.6
+VERSION = 1.1.0
```
Comment (andressbarajas) [inline Makefile.cfg]:
```suggestion
VERSION = 1.0.6
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# dcload-serial 1.0.6
+# dcload-serial 1.1.0
```
Comment (andressbarajas) [inline README.md]:
```suggestion
# dcload-serial 1.0.6
```

**Review by andressbarajas — CHANGES_REQUESTED** [review]

**Comment by Octoate** [comment]
Hi,
thanks for having a look at the PR. It took a bit from my side as well, so absolutely no problem :-). The version change was indeed for testing my version and it absolutely makes sense to not change it.
I hope that the PR is okay now.

@andressbarajas Thanks for your suggestion 👍

**Review by ljsebald — APPROVED** [review]

**Pull request merged**